filament/backend/src/opengl/OpenGLProgram.h

```diff
@@ -74,10 +74,10 @@ class OpenGLProgram : public backend::HwProgram {
     struct BlockInfo {
         uint8_t binding : 3;    // binding (i.e.: index in mSamplerBindings)
         uint8_t unused  : 1;    // padding / available
-        uint8_t count   : 4;    // number of TMUs actually used minus 1
+        uint8_t count   : 5;    // number of TMUs actually used minus 1
 
```

Comment:
```suggestion
static_assert(sizeof(BlockInfo)==1, "BlockInfo has grown. Is it intentional?");
```

Reply:
I'm not sure... but it looks better than nothing.